meetings/2025-04-09.md

```diff
@@ -0,0 +1,120 @@
+* Join queue to talk.
+* Topics can be proposed at the bottom of the agenda to be discussed as time allows. Make it known if a topic is urgent or cannot be postponed.
+
+### Participation and Code of Conduct
+* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/).
+* [Solid Code of Conduct](https://github.com/solid/process/blob/main/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/)
+* Operating principle for effective participation is to allow access across disabilities, across country borders, and across time. Feedback on tooling and meeting timing is welcome.
+* If this is your first time, welcome! please introduce yourself.
+
+
+### Scribes
+
+* elf Pavlik
+
+## Topics
+
+### Peaceful Progress
+(topic moved forward from [21 may 2025 agenda](https://hackmd.io/dtohRN8ZSVqBgRElSwEZlw))
+
+* MdJ: In the interest of our community's social harmony, let's focus our work as a CG on topics that don't overlap with topics that other groups within the Solid Community may have principled opinions about.
+  ... And it's not just about the social harmony. There is also a technical requirement. The power of Solid is that we're creating a _standard_ way for folks to run and use their personal online data stores. Creating a non-standard way to do that would be trivial, but creating a standard way requires convergence on many levels (client-server as well as client-client).
+  ... How can we achieve this convergence which is so essential for Solid? Would it for instance be better for the Solid Project as a whole if for instance Solid CG and Solid Lab Flanders would coordinate more? Or if Solid OS would coordinate more with SAI? Otherwise I fear we can continue to incubate many parallel experiments for a long time, without delivering on Solid's promise.; [JZ: and relative scope of CG compared to Practitioners]
+* eP: CGs are a bit informal, we cannot expect too much from the reports they produce. They are incubation spaces and can explore different directions. The expectaction from the LWS WG  which has more formal process is to publish specifications. So as a CG we shouldn't make too far-fetched promises about stability. We only incubate, there is zero stability. We all can support the WG to keep moving forward and to resolve objections using the established process, and that can maybe help us avoid similar situations (like what happened with PR 711) in the future.
+* JZ: In relation to WG, we will always be ahead of them. We are trying things out and we don't want to do anything that WG decided. Isn't it CG's job to experiment? This can serve as input for the WG decisions. If they decide differently we will have to reevaluate that.
```

Comment:
```suggestion
* JZ: In relation to WG, we will always be ahead of them. We are trying things out, and we don't want to do anything that WG decided. Isn't it CG's job to experiment? This can serve as input for the WG decisions. If they decide differently, we will have to reevaluate that.
```